examples/ssr-caching/pages/blog.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react'
 
-export default class extends React.Component {
-  static getInitialProps({ query: { id } }) {
-    return { id }
-  }
+export default function(props) {
+  return (
+    <div>
+      <h1>My {props.id} blog post</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua.
+      </p>
+    </div>
+  )
+}
 
-  render() {
-    return (
-      <div>
-        <h1>My {this.props.id} blog post</h1>
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua.
-        </p>
-      </div>
-    )
-  }
+export async function getServerSideProps({ query: { id } }) {
```

Comment:
This should leverage `getStaticProps` as it's a blog page.

Reply:
👍 Now it's updated with `getStaticProps`.